### ISSUE TITLE
Invalid syntax for Python One-line method example

### ISSUE THREAD

**Issue by learnbyexample** [scala/docs.scala-lang#2019]
From https://docs.scala-lang.org/scala3/book/scala-for-python-devs.html#oop-style-classes-and-methods:

>One-line method:
>`def add(a,b) = a + b`
>`def add(a: Int, b: Int): Int = a + b`

As far as I know, the above Python version would give an error. Here's what I get on version 3.9.5:

```python
>>> def add(a,b) = a + b
  File "<stdin>", line 1
    def add(a,b) = a + b
                 ^
SyntaxError: invalid syntax
```

You can use this instead:

```python
def add(a,b): return a + b
```

**Comment by casperlehmann** [comment]
There has already been submitted a PR for this: https://github.com/scala/docs.scala-lang/pull/2023

**Comment by b-studios** [comment]
Closed by #2023, thanks!
